doc/modules/ROOT/pages/rel-embedding-models.adoc

```diff
@@ -53,7 +53,7 @@ Since the prediction parts of the computations in these methods are the same, th
 | Name               | Type                       |
 | source_node_filter | Union[str, int, list[int]] | The specification of source nodes to consider. Either a node label, a node ID, or a list of node IDs
 | target_node_filter | Union[str, int, list[int]] | The specification of source nodes to consider. Either a node label, a node ID, or a list of node IDs
-| relationshipTypes  | str or list[str]           | Relationship types for which the algorithm will be executed
+| relationship_type  | str or list[str]           | Relationship types for which the algorithm will be executed
```

Comment:
its not `relationship_types`? 🤔

Reply:
Surprisingly, no 😅

Function parameter is exactly `relationship_type`

Reply:
I think it might be better to clarify the description so as not to confuse with `relationshipTypes`. This is how we describe the parameter in the reference docs: `The name of the relationship type whose embedding will be used in the computation`

Reply:
makes more sense.
shouldnt the type only be `str` and not `str or list[str]`?

Reply:
Yes, it should, thanks!